### PR TITLE
Closes #2898

### LINKED ISSUE #2898: Cannot configure fontforge with --enable-type3 option
Hi,
I'm working on the French translation of FontForge and want to enable Multi Layered Editing of Type3.  From the documentation here: http://fontforge.github.io/en-US/documentation/interface/multilayer/, it tells me to configure fontforge with the --enable-type3 option

I get this error when doing this on Ubuntu 16.04:

./configure --enable-type3
configure: WARNING: unrecognized options: --enable-type3

I'm using the master branch forked on Sept. 24.

Has type3 support been removed intentionally or Am I doing something wrong?
### Important

Mark with [x] to select. Leave as [ ] to unselect.
### When reporting a bug/issue:
- [ ] Screenshot
- [X] The FontForge version and the operating system you're using
- [X] The behavior you expect to see, and the actual behavior
- [X ] Steps to reproduce the behavior 
- [ ] (optional) Possible solution/fix/workaround

### PR DESCRIPTION
**Pull request by jrbastien**
### Motivation and Context
- [X] Why is this change required? What problem does it solve?
**Documentation is incomplete regarding conversion to multi-layers font.**
- [X] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [X] Describe your changes in detail.
**Just added to select "All layers cubic".  The configure option --enable-type3 was already removed from this page. Pull request #107 in fontforge.github.io should also be merged to keep both documentation in sync.**

### Final checklist
Go over all the following points and check all the boxes that apply. 
If you're unsure about any of these, don't hesitate to ask. We're here to help! Various areas of the codebase have been worked on by different people in recent years, so if you are unfamiliar with the general area you're working in, please feel free to chat with people who have experience in that area. See the list [here](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#people-to-ask).
- [X] My code follows the code style of this project found [here](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#coding-style).
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING**](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: doc/html/multilayer.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <H1 ALIGN=Center>
   command is placed in a separate layer and may be edited in isolation.
   <P>
   A new font may be converted to a multi-layered font by selecting
-  Element-&gt;Font Info-&gt;Layers-&gt;Multi Layered Font.
+  Element-&gt;Font Info-&gt;Layers-&gt;All layers cubic and Type3 Multi Layered Font.
```
Comment (frank-trampe) [inline doc/html/multilayer.html]: In present FontForge, the option is named Type3 Multi Layered Font.

Comment (jrbastien) [inline doc/html/multilayer.html]: What I meant by this, is that there are two options to select in order to convert the font to multi layers.  If "All layers cubic" is not selected, the "Type3 Multi Layer" option is not available.

Feel free to rewrite this, if it is unclear.
mutli-layer-options

**Comment by frank-trampe** [comment]
Ah. In that case, tell the user to navigate to that pane, to set the all layers cubic option and then to set the multi-layer option. It's confusing to refer to a non-existent hybrid option.

**Comment by jrbastien** [comment]
How about this?

  A new font may be converted to a multi-layered font by navigating to
  Element-&gt;Font Info-&gt;Layers, then setting the splines option to **All layers
  cubic** and the font type option to **Type3 Multi Layered Font**.

Let me know if that's ok with you and if I need to make another pull request.  I'm not too familar with the process.

**Comment by frank-trampe** [comment]
That's fine. It's probably easier just to make a new pull request rather than force-pushing to the same branch.

**Comment by frank-trampe** [comment]
And put the closes issue number remark in body text, not in the title. GitHub misses the automatic linkage when it is in the title.

**Comment by jrbastien** [comment]
As discuss, I will submit a new one with updated text.

**Pull request closed without merging**